### DISCUSSION TITLE
Officially Support for AndroidTV (frontend)

### DISCUSSION THREAD

**Discussion by stbeep1** [home-assistant/frontend#8371] [Other feature requests]
### I have checked those issues related to the Android TV app for HA, but all of them were closed on GitHub. Thank you! We wanted the HA app (frontend) to support AndroidTV for a long time!

**Comment by dshokouhi** [comment]
So for this request the goal is that the HA interface be easily navigable via the remote control. Currently its a bit cumbersome to move around unless you have a mouse connected but most users will have their android tv remote. We did one thing in the android app alleviate some of these issues by mapping D-Pad Down to the TAB key as that helps to navigate somewhat. However the experience for moving to teh sidebar and other areas can be improved I think. At least if we plan to keep the existing interface for the TV format.
